### DISCUSSION TITLE
Still having to look thread pitch up :(

### DISCUSSION THREAD

**Discussion by uDude** [adrianschlatter/threadlib#54] [General]
I'm a total newb.  Love this library!!!

I needed M8x1.25 bolt ... unfortunately I did the following:

bolt("M8x1.25", turns=N);

Proper error ....

I finally discovered that "M8" ==> "M8x1.25" after digging into the THREAD_TABLE scad file.

As a newb, I figured M8x1.25 were not in table.  I went to ISO, forked code, start to fix when find answer mention above.

Replicant stuff may suck/ both entry for 

"M8"
"M8x1.25" 

shold give desired result.  Replicant row in table fixes but no elegant.
Code change ugly too as search must M8x1.25 to know is just M8.

We newbs are "screwed" by believe know to this info.

Love library. Is very well design.  Have mate custom thread on big company expensive wheelchair so 3d print own cup holder.  Strokes damages language and other brain, apology please for bad grammar.  Likely forget this message sent -- hope to find email remind me.

**Comment by adrianschlatter** [comment]
Hi @uDude.

I understand you wanted to create an M8x1.25 which is listed in threadlib only under the name of "M8" but not "M8x1.25". Is this correct?

**Comment by adrianschlatter** [comment]
The development branch now list threads under their shortened designator PLUS their full designator. No matter which designator you know, threadlib will find the appropriate thread specs.
